lib/lita/handlers/jira_issues.rb

```diff
@@ -10,6 +10,8 @@ class JiraIssues < Handler
       config :password, required: true, type: String
       config :ignore, default: [], type: Array
       config :issue_ttl, default: 0, type: Integer
+      config :format, default: "[%I] %t\nStatus: %s, assigned to %a, rep. by %r, fixVersion: %v, priority: %P\n%U", type: String
```

Comment:
The test is failing because the default behaviour is to leave the priority as is from JIRA. Could you please update the default priority to be `%p` and update the specs to expect lower case?

Also, have you considered a heredoc for the default example? It might read better without the need for `\n`.